src/Tribe/RSVP.php

```diff
@@ -1685,8 +1691,8 @@ public function update_sales_by_order_status( $order_id, $attendee_order_status,
 			return;
 		}
 
-		$previous_order_status_stock_size = $rsvp_options[ $previous_order_status ]['decrease_stock_by'];
-		$attendee_order_status_stock_size = $rsvp_options[ $attendee_order_status ]['decrease_stock_by'];
+		$previous_order_status_stock_size = $rsvp_options[ $previous_order_status ][ 'decrease_stock_by' ];
+		$attendee_order_status_stock_size = $rsvp_options[ $attendee_order_status ][ 'decrease_stock_by' ];
```

Comment:
remove spaces in `[ 'decrease_stock_by' ]` but keep them around the `$attendee_order_status` variable key